File: Proiectul_meu/Models/Trening.cs
using Proiectul_meu.Models.Base;
using System.ComponentModel.DataAnnotations.Schema;

namespace Proiectul_meu.Models
{
    public class Trening : Baza
    {

        [ForeignKey("BluzaId")]
        public Bluza Bluza { get; set; }

        public Guid BluzaId { get; set; }

        public ICollection<TricouLaTrening>? Tricouri { get; set; }

        [ForeignKey("PantaloniId")]
        public Pantaloni Pantaloni { get; set; }

        public Guid PantaloniId { get; set; }
    }
        
}

File: Proiectul_meu/ClientApp/src/app/models/Trening.ts
import { Bluza } from "./Bluza";
import { Pantaloni } from "./Pantaloni";
import { TricouLaTrening } from "./TricouLaTrening";

export interface Trening {
  id?: string;
  bluza: Bluza;
  bluzaId?: string;
  tricouri?: Array<TricouLaTrening>;
  pantaloni: Pantaloni;
}
